driver-core/src/main/com/mongodb/internal/operation/QueryBatchCursor.java

```diff
@@ -60,9 +62,11 @@
 import static com.mongodb.internal.operation.OperationHelper.getMoreCursorDocumentToQueryResult;
 import static com.mongodb.internal.operation.QueryHelper.translateCommandException;
 import static com.mongodb.internal.operation.ServerVersionHelper.serverIsAtLeastVersionThreeDotTwo;
+import static java.lang.String.format;
 import static java.util.Collections.singletonList;
 
 class QueryBatchCursor<T> implements AggregateResponseBatchCursor<T> {
+    private static final Logger LOGGER = Loggers.getLogger("operation");
```

Comment:
`AsyncQueryBatchCursor` uses `com.mongodb.internal.operation.OperationHelper.LOGGER`. For consistency, we should either use the same object here, or create its own instance of a logger in `AsyncQueryBatchCursor`.

Reply:
> For consistency

What is being kept consistent?  I don't think it's a problem to repeat loggers like this.  We do it all over the place. (search for `Loggers.getLogger` and you'll see).

> or create its own instance of a logger in AsyncQueryBatchCursor

What do you mean?  With a different name? I'm trying not to introduce a different name since we document all the logger name that we use.

Reply:
> What is being kept consistent?
The way `AsyncQueryBatchCursor` and `QueryBatchCursor` access a logger object. There is no `AsyncQueryBatchCursor.LOGGER`, instead this class uses `com.mongodb.internal.operation.OperationHelper.LOGGER`. `QueryBatchCursor` is its synchronous counterpart, so it should also use `com.mongodb.internal.operation.OperationHelper.LOGGER` instead of having `QueryBatchCursor. LOGGER`.

> or create its own instance of a logger in AsyncQueryBatchCursor
> With a different name?

No, I meant a different instance with the same name (though I was incorrect, because `Loggers.getLogger` does not specify whether it returns a reference to the same instance or not, though the library we use creates a new instance). Anyway, please disregard this and only pay attention to the part related to consistency of the code:)

Reply:
Oh... I just copy and pasted that line of code from sync to async, and IntelliJ auto-imported the LOGGER. I didn't even notice.  

Done